README.md

```diff
@@ -2,15 +2,22 @@
 
 Simple Node.js based application that fetches customers and products from a commercetools project.
 
+Uses commercetools [nodejs SDK](https://commercetools.github.io/nodejs/sdk/)
+
 # Setup
-First install the latest version of node to your system
-Copy your Client Id, Client Secret and Project Key from the commercetools Admin Center and insert into the config.js file.
+1. Install the latest version of node on your system
+2. Clone this repository
+3. Copy your Client Id, Client Secret and Project Key from the commercetools Admin Center
+4. Insert these strings into the config.js file
```

Comment:
you can link to a file via [config.js](./config.js).  Also your config.js is missing api host.